exercises/code-scanning.md

```diff
@@ -318,5 +318,89 @@ queries:
 ```
 </details>
 
+#### _Stretch Exercise 4a: Uploading the SARIF as a workflow artifact_
```

Comment:
love this! super cool ! 👍 

maybe we could add one line describing why we might want to obtain the sarif artifact, other than having it as the complete record of results?